Sources/TuistSupport/Constants.swift

```diff
@@ -8,7 +8,7 @@ public enum Constants {
     public static let githubAPIURL = "https://api.github.com"
     public static let githubSlug = "tuist/tuist"
     public static let communityURL = "https://github.com/tuist/tuist/discussions/categories/general"
-    public static let version = "3.19.0"
```

Comment:
revert before merging the PR

Reply:
We created our own custom Tuist version and distributed amongst our iOS developers internally. Thats why the change in Version value

Reply:
What's the purpose of having this change committed? Once you have build and distributed it it shouldn't be needed, isn't it?

Reply:
we didn't use the github actions for the release properly, so we had to manually commit this 😅